nixos/doc/manual/from_md/release-notes/rl-2205.section.xml

```diff
@@ -826,6 +826,58 @@
           include serif fonts.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          <literal>pkgs.epgstation</literal> has been upgraded from v1
+          to v2, resulting in incompatible changes in the database
+          scheme and configuration format.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Some top-level settings under
+          <link linkend="opt-services.epgstation.enable">services.epgstation</link>
```

Comment:
```suggestion
          <link linkend="opt-services.epgstation.enable">services.epgstation.enable</link>
```

?

Reply:
That part refers to the `services.epgstation` module. But since I couldn't directly link to a module, I opted to link to the `enable` option instead.